update-version.sh

```diff
@@ -4,7 +4,8 @@ VERSION_STRING=$1
 VERSION_ARRAY=( ${VERSION_STRING//./ } )
```

Comment:
Sorry I didn't catch this before: is it possible we could put this in the `script/` directory?